core/src/main/java/com/googlecode/psiprobe/beans/stats/providers/MultipleSeriesProvider.java

```diff
@@ -36,7 +36,7 @@
 public class MultipleSeriesProvider extends AbstractSeriesProvider {
   
   /** The stat name prefix. */
-  private String statNamePrefix;
```

Comment:
Inner class accesses the field.  If Eclipse checks are turned on, it will flag this condition wanting it to be default scoped.  Generally I try to avoid inner classes, primarily becuase they are really hard to test successfully.  I think a better fix depending on the condition is extracting the inner class.  I also try to make everything private as I find it more confusing otherwise but this is the easiest fix for now.  I think it does more to point out code smell issues more than anything else.